### PR TITLE
Code cleanup

### LINKED ISSUE #1406: [REFACTOR] Remove methods from timer stop event in Execution Manager
related to #1389 

as part of the pull request we noticed that there are a few methods inside the timer_stopped event handler. This is not so great, because there are more things to do when the timer stops, and those are actually present  in the ExecutionManager.Stop() method already.

Things to do:
1. move all method calls to stop

2. change the exception handling
There are currently only two places where the `timer.Enabled = false` is done, which stops the timer and triggers the callback.
  - in the ExecutionManager `Stop()`
  - in ExecutionManager config execution inside one of the Exception `catch` blocks.

  For the second, we simply should call Stop, instead of stopping the timer.

### PR DESCRIPTION
**Pull request by neilenns**
Fixes #1406 

* Move method calls out of the timer Stopped() event handler
* Call Stop() instead of setting the timer's state via a property

### CODE REVIEW

**Comment by neilenns** [comment]
Is this what you were thinking @DocMoebiuz ?

File: MobiFlight/ExecutionManager.cs [hunk 1]
```diff
@@ -1023,7 +1022,7 @@ void timer_Tick(object sender, EventArgs e)
             {
                 Log.Instance.log($"Error on config execution: {ex.Message}", LogSeverity.Error);
                 isExecuting = false;
```
Comment (DocMoebiuz) [inline MobiFlight/ExecutionManager.cs:1024]: can be removed. is done in `Stop()`

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
Exactly. The one line can also be removed.

File: MobiFlight/ExecutionManager.cs (hunk 1 above)
Comment (neilenns) [reply MobiFlight/ExecutionManager.cs]: Fixed

**Review by DocMoebiuz — APPROVED** [review]
Cool!

**Review by DocMoebiuz — APPROVED** [review]

**Pull request merged**